packages/tools/devtools/devtools-view/src/components/TelemetryView.tsx

```diff
@@ -59,7 +61,9 @@ export function TelemetryView(): React.ReactElement {
 	const [telemetryEvents, setTelemetryEvents] = React.useState<
 		ITimestampedTelemetryEvent[] | undefined
 	>();
+	const [bufferedEvents, setBufferedEvents] = React.useState<ITimestampedTelemetryEvent[]>([]);
 	const [maxEventsToDisplay, setMaxEventsToDisplay] = React.useState<number>(DEFAULT_PAGE_SIZE);
+	const [selectedIndex, setSelectedIndex] = React.useState<number>(-1);
```

Comment:
I would recommend making this of type `number | undefined` and using `undefined` to represent the lack of a selection, rather than using a "magic" value.